examples/sdkconfig.defaults

```diff
@@ -31,6 +32,7 @@ CONFIG_SPIRAM_MALLOC_ALWAYSINTERNAL=4090
 
 #CONFIG_MBEDTLS_DEBUG=y
 CONFIG_MBEDTLS_SSL_PROTO_DTLS=y
+CONFIG_MBEDTLS_DEFAULT_MEM_ALLOC=y
```

Comment:
For security reason MBEDTLS by default only allocates in the internal ram. This option lets it allocate on the SPIRAM the security risk is that someone may reads its content viw the spi bus

Reply:
Usually, I'd say that if you have physical access to a device it is probably game over already, so fine. But, in this case, since the esp32 is maybe really out there in the world, physical access is expected. On the other hand, it looks like there are options to encrypt SPIRAM on some esp32 platforms (https://www.esp32.com/viewtopic.php?t=6513 and https://docs.espressif.com/projects/esp-idf/en/stable/esp32s3/api-guides/external-ram.html#encryption) and use eFuse to disable JTAG.

Let's say that wasn't done, and someone did pull TLS info out via SPI. What would that permit?

Reply:
they could get the client random and the session key thus decrypting all tls packets potentially for either the webrtc connection or the app connection. We could mitigate this with SPIRAM encryption but not all esp32 chip supports it